fanficfare/adapters/adapter_scribblehubcom.py

```diff
@@ -0,0 +1,280 @@
+        except HTTPError as e:
+            if e.code == 404:
+                raise exceptions.StoryDoesNotExist(self.url)
+            else:
+                raise e
+
+        if self.needToLoginCheck(data):
+            # need to log in for this one.
+            self.performLogin(url)
+            data = self._fetchUrl(url)
+
+        m = re.search(r"'viewstory.php\?sid=\d+((?:&amp;ageconsent=ok)?&amp;warning=\d+)'",data)
+        if m != None:
+            if self.is_adult or self.getConfig("is_adult"):
+                # We tried the default and still got a warning, so
+                # let's pull the warning number from the 'continue'
+                # link and reload data.
+                addurl = m.group(1)
+                # correct stupid &amp; error in url.
+                addurl = addurl.replace("&amp;","&")
+                url = self.url+'&index=1'+addurl
+                logger.debug("URL 2nd try: "+url)
+
+                try:
+                    data = self._fetchUrl(url)
+                except HTTPError as e:
+                    if e.code == 404:
+                        raise exceptions.StoryDoesNotExist(self.url)
+                    else:
+                        raise e
+            else:
+                raise exceptions.AdultCheckRequired(self.url)
+
+        if "Access denied. This story has not been validated by the adminstrators of this site." in data:
+            raise exceptions.AccessDenied(self.getSiteDomain() +" says: Access denied. This story has not been validated by the adminstrators of this site.")
+
+        # use BeautifulSoup HTML parser to make everything easier to find.
+        soup = self.make_soup(data)
+        # print data
+
+        # Now go hunting for all the meta data and the chapter list.
+
+        ## Title
+        pagetitle = soup.find('div',{'class':'fic_title'})
+        self.story.setMetadata('title',stripHTML(pagetitle))
+
+        # Find authorid and URL from main story page 
+        self.story.setMetadata('authorId',stripHTML(soup.find('span',{'class':'auth_name_fic'})))
+        self.story.setMetadata('authorUrl',soup.find('div',{'class':'author'}).find('div',{'property':'author'}).find('span',{'property':'name'}).find('a').get('href'))
+        self.story.setMetadata('author',stripHTML(soup.find('span',{'class':'auth_name_fic'})))
+
+        # Find the chapters:
+        # This is where scribblehub is gonna get a lil bit messy..
+
+        # Get the contents list from scribblehub, iterate through and add to chapters
+        # Can be fairly certain this will not 404 - we know the story id is valid
+        # Also, fails when tested with fewer than 2 chapters, which is stumping me - but it has nothing to do with this class I think
+        contents_payload = "action=wi_gettocchp&strSID=" + self.story.getMetadata('storyId') + "&strmypostid=0&strFic=yes"     
+        contents_data = self._postUrl_raw("https://www.scribblehub.com/wp-admin/admin-ajax.php", contents_payload)
```

Comment:
Instead of adding a 'raw' method, what's wrong with calling `_postUrl()` with the `dict()` parameter it expects instead of a string?

This works for me:
```
        contents_payload = {"action":"wi_gettocchp",
                            "strSID":self.story.getMetadata('storyId'),
                            "strmypostid":0,
                            "strFic":"yes"}
        contents_data = self._postUrl("https://www.scribblehub.com/wp-admin/admin-ajax.php", contents_payload)
```

Reply:
Wow, now I feel silly haha, I tried this in #507 - it's commented out on line 91 `# payload = {"action": "wi_gettocchp", "strSID": self.story.getMetadata('storyId'), "strmypostid": "0", "strFic": "yes"}`... but I passed "strmypostid" as a string not an int 😔

Reply:
The PR has picked up your changes already, it looks like.

To be honest, I paid no attention whatsoever to int vs string there--I just left it a `0` because it was.... `0`.  I'd have assumed `'0'` to be treated the same.